tensorflow-core/tensorflow-core-native/src/main/java/org/tensorflow/internal/c_api/presets/tensorflow.java

```diff
@@ -36,9 +36,11 @@
           value = {"linux", "macosx", "windows"},
           compiler = "cpp17",
           include = {
-            "tensorflow/tsl/platform/ctstring_internal.h",
-            "tensorflow/tsl/platform/ctstring.h",
-            "tensorflow/tsl/c/tsl_status.h",
+            // TSL headers are in different places in a bazel build and the downloaded whl
+            // The lower part is still the same, so multiple roots are set in the pom file.
+            "tsl/platform/ctstring_internal.h",
+            "tsl/platform/ctstring.h",
+            "tsl/c/tsl_status.h",
```

Comment:
I don't see a root in the pom file that has the `tensorflow/` suffix, to replace the existing paths, is this still required?